python/cuspatial/cuspatial/core/binops/distance_dispatch.py

```diff
@@ -0,0 +1,190 @@
+import cudf
+from cudf.core.column import arange, full
+
+from cuspatial._lib.distance import (
+    pairwise_linestring_distance,
+    pairwise_linestring_polygon_distance,
+    pairwise_point_distance,
+    pairwise_point_linestring_distance,
+    pairwise_point_polygon_distance,
+    pairwise_polygon_distance,
+)
+from cuspatial._lib.types import CollectionType
+from cuspatial.core._column.geometa import Feature_Enum
+from cuspatial.utils.column_utils import (
+    contains_only_linestrings,
+    contains_only_multipoints,
+    contains_only_points,
+    contains_only_polygons,
+)
+
+# Maps from type combinations to a tuple of (function, reverse,
+# point_collection_types) where if reverse is True, the arguments
```

Comment:
Can you document the condition under which reverse is True?

Reply:
done